site/es/blog/new-in-devtools-105/index.md

```diff
@@ -29,70 +28,89 @@ _Gracias  por la traducción [Carlos Caballero](https://carloscaballero.io) y  p
 
 
 <!-- ## Step-by-step replay in the Recorder {: #recorder } -->
+## Reproducción paso a paso en la grabadora {: #recorder }
 
 <!-- You can now set a breakpoint and replay a user flow step by step in the **Recorder** panel. -->
+Ahora puede establecer un punto de ruptura y reproducir un flujo de usuario paso a paso en el panel de **Recorder**.
```

Comment:
"punto de ruptura" is a valid translation for "breakpoint" but, for consistency, let's stick to the translation for the word used in other places of DevTools ("punto de interrupción"):
```suggestion
Ahora puede establecer un punto de interrupción y reproducir un flujo de usuario paso a paso en el panel de **Recorder**.
```